examples/bench_dtlz.py

```diff
@@ -0,0 +1,51 @@
+# Setup info-level logging
+import logging
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s - %(levelname)s - %(filename)s:%(funcName)s - " + \
+           "%(message)s",
+    force=True,
+)
+
+# Set DTLZ problem environment variables
+import os
+os.environ["DEEPHYPER_BENCHMARK_NDIMS"] = "5" # 5 vars
+os.environ["DEEPHYPER_BENCHMARK_NOBJS"] = "3" # 2 objs
+os.environ["DEEPHYPER_BENCHMARK_DTLZ_PROB"] = "2" # DTLZ2 problem
+os.environ["DEEPHYPER_BENCHMARK_DTLZ_OFFSET"] = "0.6" # [x_o, .., x_d]*=0.6
+
+# Load DTLZ benchmark suite
+import deephyper_benchmark as dhb
+# dhb.install("DTLZ")
```

Comment:
if nothing is required to be installed we can remove the `install` instruction I guess.

Reply:
OK